keras_core/ops/core.py

```diff
@@ -374,3 +375,18 @@ def convert_to_numpy(x):
         # We trigger it rather than duplicate it here.
         return np.array(x)
     return backend.convert_to_numpy(x)
+
+
+@keras_core_export("keras_core.ops.cond")
+def cond(pred, true_fn, false_fn):
+    """Conditionally applies `true_fn` or `false_fn`.
+
+    Args:
+        pred: Boolean scalar type
+        true_fn: A callable
```

Comment:
"Callable returning the output for the `pred == True` case."
""Callable returning the output for the `pred == False` case."